interfaces/api/display/I3DPointsViewer.h

```diff
@@ -60,19 +60,34 @@ class XPCF_CLIENTUUID("423cd9a6-f93a-4e1b-88f0-0762c1db5db3") XPCF_SERVERUUID("1
 
 
     /// @brief Display in a windows the 3D point cloud as well as the current camera, and optionnally, the previous frames and keyframes.
-    /// @param[in] points set of 3D points to display in the 3D viewer.
+    /// @param[in] pointCloud a point cloud to display in the 3D viewer.
     /// @param[in] pose poses of the current camera (transform of the camera defined in world coordinate system).
     /// @param[in] keyframesPoses (optional), poses of a set of keyframes (transform of the camera defined in world corrdinate system).
     /// @param[in] framePoses (optional), poses of a set of frames (transform of the camera defined in world corrdinate system).
-    /// @param[in] points2 (optional), a second set of 3D points to display in the 3D viewer (useful to visualize result of a bundle adjustment).
+    /// @param[in] pointCloud2 (optional), a second point cloud to display in the 3D viewer (useful to visualize result of a bundle adjustment).
     /// @param[in] keyframesPoses2 (optional), a second set of keyframes poses (transform of the camera defined in world corrdinate system, useful to visualize result of a bundle adjustment).
     /// @return FrameworkReturnCode::_SUCCESS if the window is created, else FrameworkReturnCode::_ERROR_
-    virtual FrameworkReturnCode display(const SRef<SolAR::datastructure::PointCloud> points,
+    virtual FrameworkReturnCode display(const SRef<SolAR::datastructure::PointCloud> pointCloud,
```

Comment:
Overload this method with a signature with no point cloud that will use the static point clouds in buffer (will use empty point cloud if buffer is not called).
This will avoid the burden for the user to provide empty point clouds for every calls.

Reply:
Done.